pootle/core/views.py

```diff
@@ -632,11 +634,21 @@ def get_context_data(self, *args, **kwargs):
         ctx, cookie_data = self.sidebar_announcements
         ctx.update(
             super(PootleBrowseView, self).get_context_data(*args, **kwargs))
+
+        project_code = self.project.code if self.project else None
```

Comment:
hmm... yes I'd be easier. I think I'll change.
But I wonder why `self.language` can be `Language` either `dict(code='lang code', name='Lang name')`.